src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -86,22 +93,46 @@ class TestCaseDisplayService(private val project: Project) {
             // fix Windows line separators
             val testCodeFormatted = testCode.replace("\r\n", "\n")
 
+            // Add checkbox
             val checkbox = JCheckBox()
             checkbox.isSelected = true
             testCasePanel.add(checkbox, BorderLayout.WEST)
 
+            // Toggle coverage when checkbox is clicked
             checkbox.addItemListener {
                 project.messageBus.syncPublisher(COVERAGE_SELECTION_TOGGLE_TOPIC)
                     .testGenerationResult(testName, checkbox.isSelected, editor)
             }
 
+            // Add editor
             val document = EditorFactory.getInstance().createDocument(testCodeFormatted)
             val textFieldEditor = EditorTextField(document, project, JavaFileType.INSTANCE)
-
             textFieldEditor.setOneLineMode(false)
-
             testCasePanel.add(textFieldEditor, BorderLayout.CENTER)
 
+            // Add top buttons
+            val topButtons = JPanel()
+            topButtons.layout = FlowLayout(FlowLayout.TRAILING)
+
+            // Add "Reset" button
+            val resetButton = JButton("Reset")
```

Comment:
Do you think it is feasible / possible to move the `reset` button inside the editor? I like the button itself, but it does take up a lot of vertical space on its own.